cluster/common.sh

```diff
@@ -2,6 +2,14 @@
 
 set -e
 
+function install_cnao {
+  ./cluster/kubectl.sh apply -f https://raw.githubusercontent.com/kubevirt/cluster-network-addons-operator/master/manifests/cluster-network-addons/0.35.1/namespace.yaml
```

Comment:
It would be great to use a parameter for cluster-network-addons version so we can easily update it in the future.